src/field_control_avatar.c

```diff
@@ -172,6 +182,20 @@ int ProcessPlayerFieldInput(struct FieldInput *input)
         if (TryStartStepBasedScript(&position, metatileBehavior, playerDirection) == TRUE)
             return TRUE;
     }
+
+    if (input->checkStandardWildEncounter)
+    {
+        if (input->dpadDirection == 0 || input->dpadDirection == playerDirection)
```

Comment:
Since there is no `else` inside this condition, wouldn't it make more sense to just combine them into a single check? It'd be cleaner for sure

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/0a55a7f40bd8ed14b71e6a8dfbb24da0121d83e4